ql/time/ecb.cpp

```diff
@@ -304,13 +304,6 @@ namespace QuantLib {
                 incrementAndCheckForOverlow(nextCodeStr[3]);
         }
         return nextCodeStr;
```

Comment:
I think the right fix is to remove this early return and leave the extra check below.

Reply:
Fixed